addons/survey/tests/test_survey_security.py

```diff
@@ -145,17 +145,17 @@ def test_access_survey_survey_user(self):
         self.env['survey.question'].create({'title': 'My Page', 'sequence': 0, 'is_page': True, 'question_type': False, 'survey_id': survey.id})
         self.env['survey.question'].create({'title': 'My Question', 'sequence': 1, 'survey_id': survey.id})
 
-        # Read: all
+        # Read: restricted to self or no one
```

Comment:
Does `# Create: own only` from line 143 still mean anything?
It is master, and you are rewriting most of the rules, I would expected significant changes in the tests. Let's just make sure that everything left is useful.